### ISSUE TITLE
Network problem. PowerShell ipconfig shows `Ethernet adapter vEthernet (WSL): Media disconnected` 

### ISSUE THREAD

**Issue by innns** [microsoft/WSL2-Linux-Kernel#236]
+ PowerShell 
```PowerShell
ipconfig
```
![image](https://user-images.githubusercontent.com/49517990/104084735-b1f95400-5284-11eb-90bc-e9dc58492282.png)

+ Ubuntu zsh
```bash
ifconfig
```
![image](https://user-images.githubusercontent.com/49517990/104084733-a9a11900-5284-11eb-88a0-0ebb5e1563b1.png)

When PS shows `Media disconnected` , WSL cannot connected to the windows host & the internet.

+ PowerShell
```PowerShell
ping 8.8.8.8
```
![image](https://user-images.githubusercontent.com/49517990/104084787-39df5e00-5285-11eb-920a-f0d2993fca91.png)
+ WSL
```bash
ping 8.8.8.8
```
![image](https://user-images.githubusercontent.com/49517990/104084797-59768680-5285-11eb-83ac-1ac1ddaa06cf.png)

This `Media disconnected`  problem can be triggered  by changing the Wi-Fi, locking the screen, etc.

I find that _Resetting Winsock Catalogue_ &  _Resetting Int IP_ and rebooting PC can solve it.
+ PowerShell
```PowerShell
netsh winsock reset catalog
netsh int ipv4 reset reset.log
netsh int ipv6 reset reset.log
```
then reboot.

BUT how can I avoid `Media disconnected`?
If someone has had this problem?

**Comment by PavelSosin-320** [comment]
Yes, it happens. You can check if it is the infrastructure problem using [WiFi network report](https://support.microsoft.com/en-us/windows/analyze-the-wireless-network-report-76da0daa-1db2-6049-d154-7bb679eb03ed) or WSL Virtual switch bug if the report is clean. Anyway, even in the case of network failure, Linux has services that listen and can recover Network configuration upon receiving Signals that Network is back. What I see is Network configuration lost. I think it needs attention.
